### PR TITLE
Update logic on EnableNegController.

### PR DESCRIPTION
**Pull request by sawsa307**
/assign @swetharepakula

### CODE REVIEW

File: pkg/context/context.go [hunk 1]
```diff
@@ -379,7 +381,7 @@ func (ctx *ControllerContext) Start(stopCh chan struct{}) {
 	if ctx.EnableASMConfigMap && ctx.ConfigMapInformer != nil {
 		go ctx.ConfigMapInformer.Run(stopCh)
 	}
-	if ctx.SvcNegInformer != nil {
+	if ctx.SvcNegInformer != nil && flags.F.EnableNEGController {
```
Comment (swetharepakula) [inline pkg/context/context.go:384]: is there anything else that is using the SvcNegInformer?

Comment (swetharepakula) [reply pkg/context/context.go:384]: I believe ingress also uses the svcneginformer

Comment (sawsa307) [reply pkg/context/context.go:384]: Right I missed that. It is actually being used by ingress, L4 ILB/NetLB and NEG controllers. Let me update the flag.

**Review by swetharepakula — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**